src/Pos/Delegation/Class.hs

```diff
@@ -78,13 +81,19 @@ instance Default DelegationWrap where
         msgCacheLimit = Just dlgCacheParam
         confCacheLimit = Just (dlgCacheParam `div` 5)
 
+type DelegationVar = RWVar DelegationWrap
+
+-- | Make a new 'DelegationVar'.
+--
+-- FIXME: it creates 'DelegationVar' with '_dwThisEpochPosted =
+-- HS.empty', is it even legal?
```

Comment:
Okay, I would like to move it into `mkDelegationVar` to make it harder to create `DelegationVar` and forget to initialize it.